src/views/ErrorPage/ErrorPage.scss

```diff
@@ -0,0 +1,43 @@
+.errorpage {
+
+    width: 100%;
+    height: 560px;
+    background: url('./Error.png');
+    margin-left: auto;
+    margin-right: auto;
+padding-bottom: 10px;
+
+    .ops {
+        font-family: 'Alex Brush';
```

Comment:
from what I can see, this font is used only once in your website. so I think it's better to export this text as an image))
![image](https://github.com/mikailcfrli/wine-and-dine/assets/18329725/2e44458e-3f5b-4741-8534-f21bc8b365d9)